lib/public/views/Runs/format/formatEditableNumericValue.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { formatFloat } from '../../../utilities/formatting/formatFloat.js';
+import { h } from '/js/src/index.js';
+
+/**
+ * Format a editable numeric property of run
+ *
+ * @param {Run} run run
+ * @param {string} propertyName updateable property name
+ * @param {RunDetailsModel} runDetailsModel details model
+ * @param {number} [options.inputPrecision = 0.001] precision of input value
+ * @param {number} [options.displayPrecision = 3] precision of displayed value
```

Comment:
It's strange to have one precision being 0.001 and one being 3. I would rather name it `inputDecimals` and `displayDecimals`, using then `Math.pow` to get the actual precision.

Reply:
applied